components/Card/index.js

```diff
@@ -37,7 +37,7 @@ const StyledFigure = styled.figure`
   border-radius: 1rem;
   border: 2px solid var(--color-grey);
   overflow: hidden;
-  background-color: var(--color-beige);
+  background-color: ${(props) => props.$categoryColor};
```

Comment:
This approach works well, but if you want to change the color for a category, you have to edit the entire plant data directly and this could easily lead to bugs. It would therefore be easier if the color is assigned to the corresponding category and does not have to be noted separately in the data record for each individual entry. One way to do this would be to create an object in which the category and color are linked and then use the value. 

**But** you also have your own categories data already and I think you could work with this data as it also has the information about the correct color for each category. Possible solution:

1. Add the `import` to the `categories` data.
2. Look inside of the categories for `bgcolor` by using the `plantCategory` value from the `plant` object:

```jsx
       <StyledFigure
          $categoryColor={
            categories.find(
              (category) => category.title.toLowerCase() === plant.category
            ).bgcolor
          }
        >
```
3. Remove the `categoryColor` key/value pair from your plant data. 👀 

**Note:**
The only issue I have with this approach is that the title of each category needs to be identical with the plant category. I would prefer a way where you work with an `id` or the already existing `slug` for example. But this would mean changing how you handle the categories information inside of your app for each plant: You would add for each plant object not the name of the plant category, but the unique id of each category and then you need to check for the category behind this id for each plant to display the correct name of the category. Would be a good way to do it like this. I know that you want to work with a database too, maybe this would be a good task to change this then.